testsuite/features/secondary/min_bootstrap_script.feature

```diff
@@ -1,8 +1,8 @@
 # Copyright (c) 2019-2023 SUSE LLC
 # Licensed under the terms of the MIT license.
 #
-# This feature depends on:
-# - features/secondary/min_bootstrap_ssh_key.feature : Due to the deletion of the previous SLES Minion
+# This feature can cause failures in the following features:
+# - features/secondary/min_ssh_tunnel.feature: if the minion fails to bootstrap
```

Comment:
If it's in terms of the minion failing to bootstrap, since `feature/secondary/min_ssh_tunnel.feature` runs in between, `features/secondary/min_salt_minions_page.feature` will fail if https://github.com/elariekerboull/uyuni/blob/master/testsuite/features/secondary/min_ssh_tunnel.feature#L83 does as well IMO
In other words, it would not be a direct consequence of this feature.
If it is for a different reason I might have missed it, and I can't seem to find the related card on the RRTG board, would you remember what kind of issue that was?